### PR TITLE
Update resolume-arena to 6.0.11,61108

### PR DESCRIPTION
**Pull request by brianbrownton**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by brianmorton** [comment]
```
Error: Some launch jobs were not unloaded.
+ com.resolume.arena.15800
Error: Some packages were not uninstalled.
+ com.resolume.pkg.ResolumeDXV.plist
+ com.resolume.pkg.ResolumeQuickLook.plist
Error: Build failed.
```

The packages failing are expect as they are in a `zap`, but how do I do the launch job with that PID or whatever on the end?

**Comment by brianmorton** [comment]
need help with this launch control job

**Review by victorpopkov — APPROVED** [review]

**Comment by reitermarkus** [comment]
The ones listed on Travis should be added as `pkgutil`.

**Comment by brianmorton** [comment]
Per travis, I added the `uninstall quit:` block. As discussed in the other resolume cask PR, the zap is there for shared entries.

**Comment by brianmorton** [comment]
Unless somebody wants to continue working on these packages, I will probably just close this PR and give up on this cask.

File: Casks/resolume-arena.rb [hunk 1]
```diff
@@ -1,16 +1,18 @@
 cask 'resolume-arena' do
-  version '6.0.10'
-  sha256 'e7279853a3605fdc8d426f0ad2ded4ee4dcdfb968cb8fd30aae6ee621de46231'
+  version '6.0.11,61108'
+  sha256 '9d9498dd9f4fb844b88e603399df3581367ac19a039eb4e4fd06c1884c411e35'
 
-  url "https://resolume.com/download/Resolume_Arena_#{version.major_minor_patch.dots_to_underscores}_Installer.dmg"
+  url "https://resolume.com/download/Resolume_Arena_#{version.major_minor_patch.dots_to_underscores}_rev_#{version.after_comma}_Inst$
```
Comment (vitorgalvao) [inline Casks/resolume-arena.rb]: Something went wrong here. Everything else looks fine.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Pull request merged**